src/plugin/Manager.h

```diff
@@ -427,6 +436,22 @@ class Manager {
      */
     void HookUnprocessedPacket(const Packet* packet) const;
 
+    /**
+     * Hook for intercepting event publish operations.
```

Comment:
Suggest to add some additional context here that this refers to cluster  communication,